### PR TITLE
Update web build

### LINKED ISSUE #47: building for web errors
Seems like the web build is super outdated. I had a go building it with

`node node_modules/gulp/bin/gulp
`
Then got these errors
```
ERROR in ./target/runner.js
Module not found: Error: Cannot resolve 'file' or 'directory' ./parser/parser.js in E:\DEV\bondage.js\target
 @ ./target/runner.js 7:13-42

ERROR in ./target/runner.js
Module not found: Error: Cannot resolve 'file' or 'directory' ./parser/nodes.js in E:\DEV\bondage.js\target
 @ ./target/runner.js 10:16-44
```

Is this something I am doing wrong or a genuine bug?
It spits up a new minified file, but it's only a third of the size of what is in dist to start with. I guess the error interrupted it ?

### PR DESCRIPTION
**Pull request by jhayley**
Fixes #47?

### CODE REVIEW

**Comment by blurymind** [comment]
hehe, I had a feeling it will be something like that =) thank you

btw, can you add a run script,to make it easier to build with` node run build ` :)

**Comment by blurymind** [comment]
I can submit a pr for that if you want to

**Comment by jhayley** [comment]
No prob, added it also

**Comment by blurymind** [comment]
sweet! Thank you @jhayley will test it now :) 👍

File: package.json [hunk 1]
```diff
@@ -10,6 +10,9 @@
   "bin": {
     "bondage": "src/bin/bondage.js"
   },
+  "scripts": {
```
Comment (blurymind) [inline package.json]: The build works as a charm now. Btw you already have an entry for scripts in package.json on line 6

Comment (jhayley) [reply package.json]: lol thanks 🤦‍♀

**Pull request merged**